packages/help-center/src/hooks/use-content-filter.ts

```diff
@@ -61,6 +61,23 @@ export const useContentFilter = ( node: HTMLDivElement | null ) => {
 					};
 				},
 			},
+			{
```

Comment:
```suggestion

			/**
			 * Fix width of Videopress embeds.
			 */
			{
```

Just for consistency 😃